config/manager/manager.yaml

```diff
@@ -1,3 +1,25 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: system #TODO: add the prefix here manually, next version of kustomize won't prefix namespaces.
```

Comment:
Suggest using a variable for the namespace here. It'll allow us to change the value of the variable and let it take effect in all the places instead of changing it in multiple places. Maybe in `envsubst`

Reply:
That might complicate things a little, we'd need a template specific for the namespace.